src/d3a/models/market/market_structures.py

```diff
@@ -352,9 +352,9 @@ def _to_csv(self):
 
 class MarketClearingState:
     def __init__(self):
-        self.cumulative_offers = dict()  # type: Dict[DateTime, dict()]
```

Comment:
I think the type info does not hurt to have, however it would be great to homogenize the constructors. `self.clearing = dict()` would be better IMO.